scripts/generate_docs.py

```diff
@@ -128,14 +130,15 @@ def _generate_valid_rst(fin, fout, namespace, tags, ver, rev, meta):
                     symbol_type = _find_symbol_type(symbol, meta)
                     if not symbol_type:
                         print("%s(%s) : error : symbol '%s' not found"%(fin, iline+1, symbol))
-                        continue
+                        raise Exception()
```

Comment:
[Suggestion] maybe we could `continue` here and mark a flag to fail, then check that flag and raise and exception if this flag was set that there is failure. This way we print all symbol problems

Reply:
I'll see if that's feasible.

Reply:
Done and this is a much nicer user experience.